README.md

```diff
@@ -32,29 +50,7 @@ You need to install at least one of the following options for builders:
     
 ### Serving
 
-You need to install at least one of the following options for the serving component:
-
-- Currently, OpenFunction Serving relies on Knative, so you need to [install Knative Serving](https://knative.dev/docs/install/).
-- Another Serving runtime Dapr + KEDA will be supported soon.
-
-### Tekton and Knative
-
-You can deploy Tekton and Knative follow this command.
-
-```bash
-sh hack/deploy-tekon-and-knative.sh
-```
-You deploy Tekton and Knative follow this command if you having poor network connectivity to GitHub/Googleapis.
-
-```bash
-sh hack/deploy-tekon-and-knative.sh --poor-network
-```
-
-If you want to use NodePort to expose the Tekton dashboard service, follow this command.
-
-```bash
-sh hack/deploy-tekon-and-knative.sh --tekton-dashboard-nodeport <port>
-```
+Currently, OpenFunction supportes tow runtime, Knative and OpenFuncAsync, you need to install at least one of these.
```

Comment:
Currently, OpenFunction supportes tow runtime, Knative and OpenFuncAsync, you need to install at least one of these.
=>
Currently, OpenFunction supports two serving runtimes, Knative and OpenFuncAsync. At least one of these runtimes needs to be installed.